api/caseWorkerDetailsRouter/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from 'express'
+import * as multer from 'multer'
+import { getErrorResponse, getPartialSuccess, getSuccess } from './mock-error-scenarios'
+
+const upload = multer({ dest: 'uploads/' })
+
+async function caseWorkerDetailsRoute(req: express.Request, res: express.Response) {
+    const mockErrorRes = getErrorResponse(req, res)
+    
+    if(mockErrorRes) {
+        res.status(mockErrorRes.status).json(mockErrorRes.json)
+        return
+    }
+
+    const partialSuccessRes = getPartialSuccess(req, res)
+    if(partialSuccessRes) {
+        res.send(partialSuccessRes)
+        return
```

Comment:
This is redundant. Nothing happens afterwards to warrant a return at this point.